src/routes/(app)/new/[id]/data/+page.svelte

```diff
@@ -35,10 +35,13 @@
 			<p class="mb-2 text-gray-active">2 / 2</p>
 			<img src="/logo.svg" alt="logo" class="h-4" />
 		</div>
-		<h1>Example Inputs</h1>
-		<div class="mb-2 flex flex-col gap-2">
-			<p>To improve your prompt we'll need some example inputs</p>
-			<p>These are the questions or statements you want your model to respond to.</p>
+		<h1>Initial Test Cases</h1>
+		<div class="mb-2 flex max-w-xl flex-col gap-2">
+			<p>
+				Let's create some test cases for your prompt. We've generated a few examples for you, but
+				feel free to add your own.
+			</p>
+			<p>Again, don't stress, you can add or edit test cases later!</p>
```

Comment:
Something old but it's been bugging me. If you delete all Instances it says generating.

Reply:
lol good catch